radis/test/lbl/test_broadening.py

```diff
@@ -938,6 +938,57 @@ def test_noneq_continuum(plot=False, verbose=2, warnings=True, *args, **kwargs):
     assert res < 5.2e-6
 
 
+def test_broadening_chunksize_eq(verbose=True, plot=False, *args, **kwargs):
+    """
+    Test an equilibrium spectrum with and without chunksize,
+    using different optimization strategies
+    and ensure that the residual is very small.
+
+    """
+    sf = SpectrumFactory(
+        wavelength_min=4000,
+        wavelength_max=4500,
+        cutoff=1e-27,
+        pressure=1,
+        isotope="1,2",
+        truncation=5,
+        neighbour_lines=5,
+        path_length=0.1,
+        mole_fraction=1e-3,
+        medium="vacuum",
+        optimization=None,
+        chunksize=None,
+        wstep=0.001,
+        verbose=False,
+    )
+    sf.load_databank("HITEMP-CO")
```

Comment:
Use `sf.fetch_databank("hitemp")`;  adding `molecule='CO'` in SpectrumFactory parameters
--> it will ensure it downloads the database if not there already